internal/controller/controller.go

```diff
@@ -97,7 +97,7 @@ func (c *Controller) Init() error {
 
 	c.tcpStateTable = &k8s.State{Table: make(map[int]*k8s.ServiceWithPort)}
 
-	c.deployLog = NewDeployLog()
```

Comment:
Isn't `1000` a bit much for only debugging purposes ?

Reply:
Depends on the cluster size I guess?

Reply:
I figured this could be configurable if someone wanted to contribute and fiddle with the templates in the future.